### PR TITLE
findBLIS should check local build/dep

### PR DESCRIPTION
**Pull request by m-kim**
install.sh will install blis in build/dep directory if it's not found, so findBLIS.cmake should look there as well.

### CODE REVIEW

File: cmake/findBLIS.cmake [hunk 1]
```diff
@@ -39,6 +39,9 @@ elseif(EXISTS      "${BUILD_DIR}/deps/blis/lib/libblis.a")
 elseif(EXISTS      "/usr/local/lib/libblis.a")
     set( BLIS_LIB /usr/local/lib/libblis.a )
     set( BLIS_INCLUDE_DIR /usr/local/include/blis )
+elseif(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/../deps/blis")
```
Comment (NaveenElumalaiAMD) [inline cmake/findBLIS.cmake]: Does Line-36 (elseif(EXISTS "${BUILD_DIR}/deps/blis/lib/libblis.a")) not do this?

Comment (m-kim) [reply cmake/findBLIS.cmake]: It doesn't. Where is BUILD_DIR set?

**Review by babakpst — DISMISSED** [review]

File: cmake/findBLIS.cmake (hunk 1 above)
Comment (NaveenElumalaiAMD) [reply cmake/findBLIS.cmake]: If the `${BUILD_DIR}` is not set then we could replace all `${BUILD_DIR}`  with `${CMAKE_CURRENT_BINARY_DIR}/..` that should fix the issue as well.

File: cmake/findBLIS.cmake [hunk 2]
```diff
@@ -30,12 +30,12 @@ elseif(EXISTS      "/opt/AMD/aocl/aocl-linux-aocc-4.1.0/aocc/lib_ILP64/libblis-m
...
-elseif(EXISTS      "${BUILD_DIR}/deps/amd-blis/lib/ILP64/libblis-mt.a") # 4.0 and 4.1.0
-    set( BLIS_LIB ${BUILD_DIR}/deps/amd-blis/lib/ILP64/libblis-mt.a )
-    set( BLIS_INCLUDE_DIR ${BUILD_DIR}/deps/amd-blis/include/ILP64 )
-elseif(EXISTS      "${BUILD_DIR}/deps/blis/lib/libblis.a")
-    set( BLIS_LIB ${BUILD_DIR}/deps/blis/lib/libblis.a )
-    set( BLIS_INCLUDE_DIR ${BUILD_DIR}/deps/blis/include/blis )
+elseif(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/../deps/amd-blis/lib/ILP64/libblis-mt.a") # 4.0 and 4.1.0
+    set( BLIS_LIB ${CMAKE_CURRENT_BINARY_DIR}/../deps/amd-blis/lib/ILP64/libblis-mt.a )
+    set( BLIS_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/../deps/amd-blis/include/ILP64 )
+elseif(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/../deps/blis")
```
Comment (NaveenElumalaiAMD) [inline cmake/findBLIS.cmake]: `/lib/libblis.a` is missing in the search path. This is to check before setting the path to `BLIS_LIB`

Comment (m-kim) [reply cmake/findBLIS.cmake]: Fixed.

**Review by NaveenElumalaiAMD — APPROVED** [review]
LGTM!

**Review by jichangjichang — APPROVED** [review]
lgtm

**Pull request merged**